src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -352,6 +351,9 @@ public Builder runDevAppServerWait(int runDevAppServerWaitSeconds) {
 
     /**
      * Create a new instance of {@link CloudSdk}.
+     *
```

Comment:
If you run "mvn clean install" or "mvn checkstyle:check" locally, it will do the checkstyle checking.